### PR TITLE
added schema file for Tag

### PR DESCRIPTION
**Pull request by hmmtrifork**

### CODE REVIEW

File: openapi/schemas/Tag.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type: "array"
```
Comment (joratrifork) [inline openapi/schemas/Tag.yaml:1]: Do we want this to be an array - what if we introduce a new endpoint later that only handles a single tag?

**Review by joratrifork — APPROVED** [review]
Also check for breaking changes in the engine (In case the Kotlin plugin decides to give tags a proper name)

File: openapi/schemas/Tag.yaml (hunk 1 above)
Comment (hmmtrifork) [reply openapi/schemas/Tag.yaml:1]: Good point. it will not be an array in the Tag.yaml specification

**Comment by hmmtrifork** [comment]
> Also check for breaking changes in the engine (In case the Kotlin plugin decides to give tags a proper name)

I will add an issue in the trireg2 engine repo, that the next time the submodule is updated to the latest API version, check for these breaking changes and fix them and update the API to the latest version in the same commit.
The same with the CLI repo.

**Pull request merged**